### PR TITLE
Feature milestone-2

### PR DESCRIPTION
**Pull request by Jarfsoft**
**Feature added:**

- Book object creation that include:
    - Book ID
    - Title
    - Category
- Add book
- Remove book

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello Team 👋

**You've done pretty well here, redux actions and reducers correctly configured, with redux store properly setup. 👌**

However, let's make a couple of changes.

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,13 @@
 import React from 'react';
 
+import BooksList from './components/BooksList';
```
Comment (mikenath223) [inline src/App.js]: - [x] Kindly move `App.js` into the components folder.

File: src/App.js [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
+import BooksForm from './components/BooksForm';
+
 const App = () => (
   <div>
     <h1>Hello</h1>
+    <BooksForm />
```
Comment (mikenath223) [inline src/App.js]: - [x] Please remove the Hello word as it bears no purpose here.

**Review by afanimartin — APPROVED** [review]
## Approved 👏🏿 👏🏿 👏🏿 
Nice work making the requested changes. `App.js` file move to `components` folder as requested. Merge and proceed to the next project.

### Martin Afani
#### Slack: whiz25

**Pull request merged**